ca_driver/src/create_driver.cpp

```diff
@@ -255,6 +263,16 @@ void CreateDriver::publishOdom()
   odom_pub_.publish(odom_msg_);
 }
 
+void CreateDriver::publishJointState() {
+    // Publish joint states
+    joint_state_msg_.header.stamp = ros::Time::now();
+    joint_state_msg_.position[0] = robot_->getLeftWheelDistance() / (create::util::CREATE_2_WHEEL_DIAMETER / 2);
```

Comment:
Actually, you might be right, because the constant is in a different library so the compiler can't guarantee that its value can't change. Either way, I'll just do what you suggested.